### PR TITLE
feat: add DataDeliveryVote tx and query cli

### PR DESCRIPTION
**Pull request by audtlr24**
- add `DataDeliveryVote` to `grpc_query_client.go`
- add tx & query cli for `DataDeliveryVote`

### CODE REVIEW

File: x/datadeal/client/cli/txDataDeliveryVote.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/gogo/protobuf/jsonpb"
+	"github.com/medibloc/panacea-core/v2/x/datadeal/types"
+	"github.com/spf13/cobra"
+)
+
+func CmdVoteDataDelivery() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "data-verification-vote [path]",
+		Short: "Vote for data verification",
```
Comment (inchori) [inline x/datadeal/client/cli/txDataDeliveryVote.go:17]: Typo error in `Short`.

Comment (audtlr24) [reply x/datadeal/client/cli/txDataDeliveryVote.go]: Ah, Thanks! 😸

**Review by H4NLee — APPROVED** [review]
lgtm

**Review by gyuguen — COMMENTED** [review]
LGTM

**Pull request merged**